src/read.py

```diff
@@ -0,0 +1,69 @@
+import re
```

Comment:
Using `read.py` is kind of vague. Maybe something like `file_readers.py` would be better. Filenames should describe as succinctly and as informatively as possible what the objects inside that file do.

Reply:
Renamed to `file_readers.py` (and refactored other files to follow).